src/server/handlers/api.go

```diff
@@ -150,6 +150,77 @@ func (h APIHhandler) GetMetrics(w http.ResponseWriter, r *http.Request, argv map
 	return err
 }
 
+// GetExports return a list of metrics if prmetheus export format
+func (h APIHhandler) GetExports(w http.ResponseWriter, r *http.Request, argv map[string]string) error {
+	var res []storage.Item
+	var first bool
+	var err error
+
+	// get data from the form arguments
+	if err = r.ParseForm(); err != nil {
+		return err
+	}
+
+	// we only use gauges
+	if typeStr, ok := r.Form["type"]; ok && len(typeStr) > 0 && typeStr[0] != "gauge" {
+		fmt.Fprintln(w, "[]")
+		return nil
+	}
+
+	// get tenant
+	tenant := parseTenant(r)
+
+	// get a list of gauges
+	if tagsStr, ok := r.Form["tags"]; ok && len(tagsStr) > 0 {
+		tags := storage.ParseTags(tagsStr[0])
+		if !validTags(tags) {
+			return errBadMetricID
+		}
+		res, err = h.Storage.GetItemList(tenant, tags)
+	} else {
+		res, err = h.Storage.GetItemList(tenant, map[string]string{})
+	}
+
+	if err == nil {
```

Comment:
I think it's better to check if `err` is not `nil` rather than is `nil` because I don't like indentation blocks in the code.

Reply:
Thanks, refactored the logic here.

Reply:
👍 done